.github/workflows/ci.yml

```diff
@@ -49,8 +50,13 @@ jobs:
               run: composer --version
 
             - name: Composer install
+              if: ${{ matrix.php-version != '7.4' }}
               run: composer install
 
+            - name: Composer install
+              if: ${{ matrix.php-version == '7.4' }}
+              run: composer update --prefer-lowest
```

Comment:
I removed this in purpose as it is invalid usage

Reply:
oh, it's just a copy-paste from the sdk ci. What is proper usage then?

Reply:
Using own task here, not abusing default run for 7.4

Reply:
ok. thank you. Will do in separate pr.